lua/packer/luarocks.lua

```diff
@@ -176,16 +176,16 @@ local function format_luarocks_args(package)
   if type(package) ~= 'table' then return '' end
   local args = {}
   for key, value in pairs(package) do
-    if type(key) ~= "string" or not is_valid_luarock_key(key) then goto continue end
-    local luarock_key = luarocks_keys[key] and luarocks_keys[key] or key
-    if luarock_key and type(value) == 'string' then
-      table.insert(args, string.format('--%s=%s', key, value))
-    elseif key == "env" and type(value) == "table" then
-      for name, env_value in pairs(value) do
-        table.insert(args, string.format('%s=%s', name, env_value))
+    if type(key) == "string" or is_valid_luarock_key(key) then
```

Comment:
I think this should be an `and` rather than an `or`, right? We want to process keys which are both a string (ensuring that `is_valid_luarock_key` will behave as expected) and for which `is_valid_luarock_key` returns `true`.

Reply:
Sorry that's right, pushed the change in a bit of a rush last night

Reply:
No worries and thanks again!